app/mailers/notification_mailer.rb

```diff
@@ -0,0 +1,11 @@
+class NotificationMailer < ApplicationMailer
+  @@admin = Rails.application.secrets.notification_email_recipient_address
```

Comment:
There shouldn't be a need for a class variable like this.
Since this is a constant that won't change during the life of the application, you can then usually the constants syntax like so:

`ADMIN = value`

A variable with all capital letters is a constant.